doc/architecture/decisions/0004-generate-domain-events-when-ap-data-is-changed.md

```diff
@@ -0,0 +1,128 @@
+# 4. Generate Domain Events when AP data is changed
+
+Date: 2022-10-18
+
+## Status
+
+Approved
+
+## Context
```

Comment:
Is it worth talking about alternative approaches - for example, I know we were talking about some kind of CSV format during alpha

Reply:
Do you mean alternative approaches to:

1.  **communicating changes to Delius?** We've had a very strong steer than using Domain Events is the right way to do it.  Or were you referring to 

2. **communicating metrics to the performance hub?** Maybe I should be explicit that Events won't be consumed to produce KPIs -- instead we'll generate a CSV for the performance hub by querying our stored events.